src/ToDoApp.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useCallback, useState } from "react";
+import Container from "./components/Container.tsx";
+import Input from "./components/Input.tsx";
+import Button from "./components/Button.tsx";
+
+interface Task {
```

Comment:
```suggestion
type Task {
id: UUID;
...
 }
```

Reply:
@tranvuminhtriet can you describe in details for @quixun when should we use interface, type?